### PR TITLE
[FIX] File: no domain or empty domain -> no data

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
screenshot_20170523_181635
screenshot_20170523_132525

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/278927799/

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/196743824/

File: Orange/widgets/data/owfile.py [hunk 1]
```diff
@@ -410,6 +410,10 @@ def apply_domain_edit(self):
         else:
             table = self.data
 
+        if not hasattr(table, "domain") or hasattr(table, "domain") and \
```
Comment (kernc) [inline Orange/widgets/data/owfile.py]: Could you simplify to `table is not None and ...`? As per above branching, `table` can be either a Table or None.

Comment (jerneju) [inline Orange/widgets/data/owfile.py]: Fixed.

**Pull request merged**